$NetBSD: patch-ab,v 1.1.96.1 2017/02/15 19:39:10 bsiegert Exp $

Document that escape sequence converts to "telnet break" when using telnet.

--- conserver.cf/conserver.cf.man.orig	2004-11-06 01:20:30.000000000 +0100
+++ conserver.cf/conserver.cf.man
@@ -271,7 +271,30 @@ is ``and''ed with 0x1f)
 .PD
 .RE
 .RE
+.sp
+For consoles accessed using the telnet protocol, the \f3\ez\fP
+sequence will be translated to the telnet protocol's ``send break''
+protocol sequence.
+.sp
+Conserver provides the following predefined default break strings:
+.RS
+.sp
+.PD 0
+.TP
+\f30\fP
+\ez
+.TP
+\f31\fP
+\er~^b
+.TP
+\f32\fP
+#.
+.TP
+\f33\fP
+\er\ed~\ed^b    (with 600ms delay)
 .RE
+.RE
+.sp
 .TP
 \f3config\fP \f2hostname\fP|\f2ipaddr\fP
 .br
